Post/tests.py

```diff
@@ -4,7 +4,9 @@
 class TestPost():
```

Comment:
Please note that it will not be useful to group all post tests under a class. Usually a "feature" or a behavior can be grouped, and all relevant tests will exists there.
But when the team will start writing a lot of tests, these will come in separate files to make it easier to maintain.

Just a not for now, consider this for the coming changes.